files/es/web/api/batterymanager/charging/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: BatteryManager.charging
+slug: Web/API/BatteryManager/charging
+tags:
+  - Apps
+  - Batería
+  - Battery
+  - Firefox OS
+  - Mobile
+  - Móvil
+  - WebAPI
+translation_of: Web/API/BatteryManager/charging
+browser-compat: api.BatteryManager.charging
+---
+{{deprecated_header}}{{APIRef("Battery API")}}
+
+Un valor boleano indicando si está cargando o no la batería del dispositivo (está conectado el cargador).
+
+## Sintaxis
+
+```js
+var cargando = battery.charging
```

Comment:
Que tal! creo que no traducimos el códiigo, solo los comentarios o mensajes... @Graywolf9 vdd?
```suggestion
var charging = battery.charging
```

Reply:
If I understood your concern correctly, variable names can be translated without a problem, so feel free to keep Vallejoanderson's original version :)

Reply:
Creo que en el tema de declaracion de variables podria ser beneficioso para que el aprendiz pueda diferenciarlo del objeto. Esperamos sugerencias de @Graywolf9 ;)

Reply:
yes, you understood correctly.
Okay, now we know we can do it, we will need to standardize, isn't it? and decide if we will translate or not the variable names.

Reply:
Since all variable names have been translated in this file, I think you could keep them. For other pages, it's not strictly necessary to translate every variable or function name (e.g. you'll often notice that French pages keep those in English, which is also fine) — probably something to discuss on Matrix

Reply:
@sergiogval este punto aún estaba en discusión, si lo leiste antes de hacer merge? si no lo leiste ntp, solo para saber y tener cuidado para la próxima, de hecho que genial que llegaste a mergear por que necesitabamos a alguien que lo hiciera dado que creo debemos dar dos reviews y merge, vdd?